src/packages/storage.gleam

```diff
@@ -207,11 +207,15 @@ fn hex_package_to_storage_package(
   }
   let repository_url =
     dict.get(package.meta.links, "Repository") |> option.from_result
+
+  let assert Ok(inserted_in_hex_at) = birl.parse(package.inserted_at)
+  let assert Ok(updated_in_hex_at) = birl.parse(package.updated_at)
```

Comment:
We shouldn't ever assert, but the real fix here is to make the hexpm library use `gleam_time`. Can do that before or after merge, I don't mind.

Reply:
I've updated the package and pushed a v3!